src/NewChain/Initialization.ts

```diff
@@ -180,14 +180,14 @@ export default class Initialization {
     );
     const auxiliaryContracts = auxiliaryChain.contractAddresses.auxiliary;
 
-    auxiliaryContracts.anchorOrganizationAddress = anchorOrganization.address;
-    auxiliaryContracts.anchorAddress = anchor.address;
-    auxiliaryContracts.ostCoGatewayOrganizationAddress = coGatewayAndOstPrimeOrganization.address;
-    auxiliaryContracts.ostPrimeAddress = ostPrime.address;
-    auxiliaryContracts.ostEIP20CogatewayAddress = ostCoGateway.address;
-    auxiliaryContracts.gatewayLibAddress = gatewayLib.address;
-    auxiliaryContracts.messageBusAddress = messageBus.address;
-    auxiliaryContracts.merklePatriciaLibAddress = merklePatriciaProof.address;
+    auxiliaryContracts.anchorOrganizationAddress = Web3.utils.toChecksumAddress(anchorOrganization.address);
```

Comment:
I feel that all calls to external methods should be wrapped in a method in our code. 

In this context, we are directly calling  `Web3.utils.toChecksumAddress` on `import Web3 = require('web3')`. If something inside `Web3` changes we would need to change the usages at a lot of places in our code base. 

I would propose to add a small method in one of our `Util` classes which would be called from our code base.